lib/pronto/cli.rb

```diff
@@ -14,6 +14,11 @@ def is_thor_reserved_word?(word, type)
 
     desc 'run', 'Run Pronto'
 
+    method_option :config,
+                  type: :string,
+                  default: '.pronto.yml',
+                  desc: 'Uses the specified cofnig file.'
```

Comment:
There's already a `File.exist?` check in the `to_h` method that you might be interested in... 😃